march_hardware_interface/config/march-iii/controllers.yaml

```diff
@@ -3,12 +3,17 @@ march:
     state:
       type: joint_state_controller/JointStateController
       publish_rate: 50
+    position:
+      left_hip_joint:
+        type: position_controllers/JointPositionController
+        joint: left_hip
```

Comment:
Why is there only a position controller for one joint?

Reply:
Can be removed